src/commands/code-analyzer/run.ts

```diff
@@ -1,19 +1,74 @@
-import {SfCommand} from '@salesforce/sf-plugins-core';
+import {Flags, SfCommand} from '@salesforce/sf-plugins-core';
+import {RunAction} from '../../lib/actions/RunAction';
+import {BundleName, getMessage} from '../../lib/messages';
 
-export default class Run extends SfCommand<void> {
+export default class RunCommand extends SfCommand<void> {
 	// We don't need the `--json` output for this command.
 	public static readonly enableJsonFlag = false;
-	public static readonly summary = 'summary boop'; // TODO
-	public static readonly description = 'description boop'; // TODO
-	public static readonly examples = [ // TODO
-		'example1 boop',
-		'example2 boop'
+	public static readonly summary = getMessage(BundleName.RunCommand, 'command.summary');
+	public static readonly description = getMessage(BundleName.RunCommand, 'command.description');
+	public static readonly examples = [
+		getMessage(BundleName.RunCommand, 'command.examples')
 	];
 
-	public static readonly flags = {};
+	public static readonly flags = {
+		// === Flags pertaining to targeting ===
+		workspace: Flags.string({
+			summary: getMessage(BundleName.RunCommand, 'flags.workspace.summary'),
+			char: 'w',
+			multiple: true,
+			delimiter: ',',
+			default: ['.']
```

Comment:
There is a question of whether or not we just always fill in this default... or not have a default so that later we can check to see if a value was even provided (if we want to log that we are filling in a default on their behalf). I personally think that what you have here is good... but in the old scanner we had to detect that a user didn't supply anything in order to log that we filled it in.

Reply:
Do we need to log that we defaulted to a value? If so, yeah we should revert to the old style.

Reply:
I personally don't think we need to log anything. I just bring it up because we did in the old cli. But then again that was when we had target and projectDir which are a bit different than workspace. So I say stick with what you have here.